Palace/MyBooks/MyBooksDownloadCenter.swift

```diff
@@ -435,9 +435,10 @@ import OverdriveProcessor
 
 extension MyBooksDownloadCenter {
   func deleteLocalContent(for identifier: String, account: String? = nil) {
+    let current_account: String? = account ?? AccountsManager.shared.currentAccountId
     guard let book = bookRegistry.book(forIdentifier: identifier),
-          let bookURL = fileUrl(for: identifier, account: account) else {
-      NSLog("WARNING: Could not find book to delete local content.")
+          let bookURL = fileUrl(for: identifier, account: current_account) else {
+      Log.warn(#file, "Could not find book to delete local content \(identifier)")
```

Comment:
Most of the code in this file is logging via `NSLog`, but it looks like most of the Palace code is using `Log`, so I wasn't sure which way to go for log statements here. 

Let me know what you prefer for new code @mauricecarrier7. I don't mind switching this up if its something other then `Log`.

Reply:
Log is fine, that was likely just overlooked